kapitan/cli.py

```diff
@@ -208,6 +209,14 @@ def main():
     refs_parser.add_argument('--key', '-K', help='set KMS key',
                              default=from_dot_kapitan('refs', 'key', ''),
                              metavar='KEY')
+    #
+    # for azure KMS, key is got from the other component defs
+    refs_parser.add_argument('--vault', help='azure vault url ()',
```

Comment:
I'd rename option to `--azure-vault-url` to explicitly define what it is.

Reply:
ok was thinking abt that but then also --azure-vault-key ?

Reply:
the key is re-used for other backends like awskms or gkms so it wouldn't make sense to tie it to azure only. 
But vault-url is azure specific so I'd keep that clear.
Other opinions are welcome too. :)